src/annotator/util/test/buckets-test.js

```diff
@@ -116,44 +125,32 @@ describe('annotator/util/buckets', () => {
   describe('anchorBuckets', () => {
     it('puts anchors that are above the screen into the `above` bucket', () => {
       const bucketSet = anchorBuckets(fakeAnchors);
-      assert.deepEqual(bucketSet.above.anchors, [
-        fakeAnchors[0],
-        fakeAnchors[1],
-      ]);
+      assert.deepEqual([...bucketSet.above.tags], [getTag(0), getTag(1)]);
```

Comment:
If the conversion of a `Set` to an array changes the order of the elements this test will fail.